### ISSUE TITLE
ERROR: LoadError: UndefVarError: `command_main` not defined

### ISSUE THREAD

**Issue by Branda98** [kharchenkolab/Baysor#145]
Hi, I'm trying to install Baysor in Julia's REPL on my Windows using these commands (I'm using Julia 1.10.4). However, I keep getting errors.

This is what I'm running in Julia.

```julia
using Pkg
Pkg.add(PackageSpec(url="https://github.com/kharchenkolab/Baysor.git"))
Pkg.build()

``` 

and this is what I get.

> #Stacktrace:
>   [1] pkgerror(msg::String)
>     @ Pkg.Types C:\Users\brand\AppData\Local\Programs\Julia-1.10.4\share\julia\stdlib\v1.10\Pkg\src\Types.jl:70
>   [2] (::Pkg.Operations.var"#67#74"{Bool, Pkg.Types.Context, String, Pkg.Types.PackageSpec, String})()
>     @ Pkg.Operations C:\Users\brand\AppData\Local\Programs\Julia-1.10.4\share\julia\stdlib\v1.10\Pkg\src\Operations.jl:1157
>   [3] withenv(::Pkg.Operations.var"#67#74"{Bool, Pkg.Types.Context, String, Pkg.Types.PackageSpec, String}, ::Pair{String, String}, ::Vararg{Pair{String}})
>     @ Base .\env.jl:257
>   [4] (::Pkg.Operations.var"#117#122"{String, Bool, Bool, Bool, Pkg.Operations.var"#67#74"{Bool, Pkg.Types.Context, String, Pkg.Types.PackageSpec, String}, Pkg.Types.PackageSpec})()
>     @ Pkg.Operations C:\Users\brand\AppData\Local\Programs\Julia-1.10.4\share\julia\stdlib\v1.10\Pkg\src\Operations.jl:1825
>   [5] with_temp_env(fn::Pkg.Operations.var"#117#122"{String, Bool, Bool, Bool, Pkg.Operations.var"#67#74"{Bool, Pkg.Types.Context, String, Pkg.Types.PackageSpec, String}, Pkg.Types.PackageSpec}, temp_env::String)
>     @ Pkg.Operations C:\Users\brand\AppData\Local\Programs\Julia-1.10.4\share\julia\stdlib\v1.10\Pkg\src\Operations.jl:1706
>   [6] (::Pkg.Operations.var"#115#120"{Dict{String, Any}, Bool, Bool, Bool, Pkg.Operations.var"#67#74"{Bool, Pkg.Types.Context, String, Pkg.Types.PackageSpec, String}, Pkg.Types.Context, Pkg.Types.PackageSpec, String, Pkg.Types.Project, String})(tmp::String)
>     @ Pkg.Operations C:\Users\brand\AppData\Local\Programs\Julia-1.10.4\share\julia\stdlib\v1.10\Pkg\src\Operations.jl:1795
>   [7] mktempdir(fn::Pkg.Operations.var"#115#120"{Dict{String, Any}, Bool, Bool, Bool, Pkg.Operations.var"#67#74"{Bool, Pkg.Types.Context, String, Pkg.Types.PackageSpec, String}, Pkg.Types.Context, Pkg.Types.PackageSpec, String, Pkg.Types.Project, String}, parent::String; prefix::String)
>     @ Base.Filesystem .\file.jl:766
>   [8] mktempdir(fn::Function, parent::String)
>     @ Base.Filesystem .\file.jl:762
>   [9] mktempdir
>     @ .\file.jl:762 [inlined]
>  [10] sandbox(fn::Function, ctx::Pkg.Types.Context, target::Pkg.Types.PackageSpec, target_path::String, sandbox_path::String, sandbox_project_override::Pkg.Types.Project; preferences::Dict{String, Any}, force_latest_compatible_version::Bool, allow_earlier_backwards_compatible_versions::Bool, allow_reresolve::Bool)
>     @ Pkg.Operations C:\Users\brand\AppData\Local\Programs\Julia-1.10.4\share\julia\stdlib\v1.10\Pkg\src\Operations.jl:1753
>  [11] build_versions(ctx::Pkg.Types.Context, uuids::Set{Base.UUID}; verbose::Bool)
>     @ Pkg.Operations C:\Users\brand\AppData\Local\Programs\Julia-1.10.4\share\julia\stdlib\v1.10\Pkg\src\Operations.jl:1138
>  [12] build_versions
>     @ C:\Users\brand\AppData\Local\Programs\Julia-1.10.4\share\julia\stdlib\v1.10\Pkg\src\Operations.jl:1056 [inlined]
>  [13] build(ctx::Pkg.Types.Context, uuids::Set{Base.UUID}, verbose::Bool)
>     @ Pkg.Operations C:\Users\brand\AppData\Local\Programs\Julia-1.10.4\share\julia\stdlib\v1.10\Pkg\src\Operations.jl:996
>  [14] build(ctx::Pkg.Types.Context, pkgs::Vector{Pkg.Types.PackageSpec}; verbose::Bool, kwargs::@Kwargs{io::Base.TTY})
>     @ Pkg.API C:\Users\brand\AppData\Local\Programs\Julia-1.10.4\share\julia\stdlib\v1.10\Pkg\src\API.jl:1056
>  [15] build(pkgs::Vector{Pkg.Types.PackageSpec}; io::Base.TTY, kwargs::@Kwargs{})
>     @ Pkg.API C:\Users\brand\AppData\Local\Programs\Julia-1.10.4\share\julia\stdlib\v1.10\Pkg\src\API.jl:159
>  [16] build(pkgs::Vector{Pkg.Types.PackageSpec})
>     @ Pkg.API C:\Users\brand\AppData\Local\Programs\Julia-1.10.4\share\julia\stdlib\v1.10\Pkg\src\API.jl:148
>  [17] build(; name::Nothing, uuid::Nothing, version::Nothing, url::Nothing, rev::Nothing, path::Nothing, mode::Pkg.Types.PackageMode, subdir::Nothing, kwargs::@Kwargs{})
>     @ Pkg.API C:\Users\brand\AppData\Local\Programs\Julia-1.10.4\share\julia\stdlib\v1.10\Pkg\src\API.jl:174
>  [18] build()
>     @ Pkg.API C:\Users\brand\AppData\Local\Programs\Julia-1.10.4\share\julia\stdlib\v1.10\Pkg\src\API.jl:165
>  [19] top-level scope
>     @ c:\Users\brand\.julia\JuliaProj\using Pkg.jl:3
>  [20] eval
>     @ .\boot.jl:385 [inlined]
>  [21] include_string(mapexpr::typeof(identity), mod::Module, code::String, filename::String)
>     @ Base .\loading.jl:2076
>  [22] include_string(m::Module, txt::String, fname::String)
>     @ Base .\loading.jl:2086
>  [23] invokelatest(::Any, ::Any, ::Vararg{Any}; kwargs::@Kwargs{})
>     @ Base .\essentials.jl:892
>  [24] invokelatest(::Any, ::Any, ::Vararg{Any})
>     @ Base .\essentials.jl:889
>  [25] inlineeval(m::Module, code::String, code_line::Int64, code_column::Int64, file::String; softscope::Bool)
>     @ VSCodeServer c:\Users\brand\.vscode\extensions\julialang.language-julia-1.127.2\scripts\packages\VSCodeServer\src\eval.jl:271
>  [26] (::VSCodeServer.var"#69#74"{Bool, Bool, Bool, Module, String, Int64, Int64, String, VSCodeServer.ReplRunCodeRequestParams})()
>     @ VSCodeServer c:\Users\brand\.vscode\extensions\julialang.language-julia-1.127.2\scripts\packages\VSCodeServer\src\eval.jl:181
>  [27] withpath(f::VSCodeServer.var"#69#74"{Bool, Bool, Bool, Module, String, Int64, Int64, String, VSCodeServer.ReplRunCodeRequestParams}, path::String)
>     @ VSCodeServer c:\Users\brand\.vscode\extensions\julialang.language-julia-1.127.2\scripts\packages\VSCodeServer\src\repl.jl:276
>  [28] (::VSCodeServer.var"#68#73"{Bool, Bool, Bool, Module, String, Int64, Int64, String, VSCodeServer.ReplRunCodeRequestParams})()
>     @ VSCodeServer c:\Users\brand\.vscode\extensions\julialang.language-julia-1.127.2\scripts\packages\VSCodeServer\src\eval.jl:179
>  [29] hideprompt(f::VSCodeServer.var"#68#73"{Bool, Bool, Bool, Module, String, Int64, Int64, String, VSCodeServer.ReplRunCodeRequestParams})
>     @ VSCodeServer c:\Users\brand\.vscode\extensions\julialang.language-julia-1.127.2\scripts\packages\VSCodeServer\src\repl.jl:38
>  [30] (::VSCodeServer.var"#67#72"{Bool, Bool, Bool, Module, String, Int64, Int64, String, VSCodeServer.ReplRunCodeRequestParams})()
>     @ VSCodeServer c:\Users\brand\.vscode\extensions\julialang.language-julia-1.127.2\scripts\packages\VSCodeServer\src\eval.jl:150
>  [31] with_logstate(f::Function, logstate::Any)
>     @ Base.CoreLogging .\logging.jl:515
>  [32] with_logger
>     @ .\logging.jl:627 [inlined]
>  [33] (::VSCodeServer.var"#66#71"{VSCodeServer.ReplRunCodeRequestParams})()
>     @ VSCodeServer c:\Users\brand\.vscode\extensions\julialang.language-julia-1.127.2\scripts\packages\VSCodeServer\src\eval.jl:263
>  [34] #invokelatest#2
>     @ .\essentials.jl:892 [inlined]
>  [35] invokelatest(::Any)
>     @ Base .\essentials.jl:889
>  [36] (::VSCodeServer.var"#64#65")()
>     @ VSCodeServer c:\Users\brand\.vscode\extensions\julialang.language-julia-1.127.2\scripts\packages\VSCodeServer\src\eval.jl:34

> > **in expression starting at c:\Users\brand\.julia\JuliaProj\using Pkg.jl:3**

> 

"using Pkg.jl" is what I ran below

```julia
using Pkg
Pkg.add(PackageSpec(url="https://github.com/kharchenkolab/Baysor.git"))
Pkg.build()

``` 

I don't understand how Pkg.build() is the problem. Any help would be appreciated!

And in my build log, I got an error that said "ERROR: LoadError: UndefVarError: `command_main` not defined"

here's my build log.
[build.log](https://github.com/user-attachments/files/17534838/build.log)

**Comment by VPetukhov** [comment]
Or you can [install `develop` version](https://kharchenkolab.github.io/Baysor/dev/installation/#Installing-other-versions), it's fixed there.

**Comment by VPetukhov** [comment]
The changes are now released in v0.7.1. While this problem is fixed, there are other compatibility issues with julia v1.11, which I can't fix. So now Baysor is restricted to v1.10. See [more info here](https://kharchenkolab.github.io/Baysor/dev/installation/#Install-as-a-Julia-package).
